pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -308,15 +333,15 @@ func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *opv1al
 		if labels == nil {
 			labels = map[string]string{}
 		}
-		labels[opv1alpha1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
+		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
```

Comment:
Can we remove this ference somehow if not we have to use `v1` at the function name

Reply:
I think we need to use it because `SchemeGroupVersion` type is only set in `register.go` which isn't present in shared. 
Will change to `renderObjectsV1()`